tests/test.m

```diff
@@ -0,0 +1,50 @@
+% Initialization

```

Comment:
Are we adding this matlab file instead of the python one? I think it makes more sense to verify against a tested implementation, no?